src/python/doc/wasserstein_distance_user.rst

```diff
@@ -9,10 +9,16 @@ Definition
 
 .. include:: wasserstein_distance_sum.inc
 
-Functions
----------
-This implementation uses the Python Optimal Transport library and is based on
-ideas from "Large Scale Computation of Means and Cluster for Persistence
+The q-Wasserstein distance is defined as the minimal value 
+by a perfect matching between the points of the two diagrams (+ all  
```

Comment:
"by" sounds strange, maybe "of" or "for"? If you think "by" is fine, then ok.

Reply:
I guess I accidentally removed "minimal value *achieved* by"